UnrealPlugin/README.md

```diff
@@ -216,6 +216,41 @@ a dedicated server, so you could wrap the call to ReadyForPlayers() as such at t
 #endif
 ```
 ----
+##### Set the game server port number with MPS's configuration.
+```cpp
+    FGameServerConnectionInfo ConnectionInfo = UGSDKUtils::GetGameServerConnectionInfo();
+
+    for (auto& GamePorts : ConnectionInfo.GamePortsConfiguration) {
+        if (GamePorts.Name == TEXT("<portname>")) {
+            UE_LOG(LogTemp, Log, TEXT("GSDK Game port name: %s"), *GamePorts.Name);
+            UE_LOG(LogTemp, Log, TEXT("GSDK Game server listening port: %d"), GamePorts.ServerListeningPort);
+            UE_LOG(LogTemp, Log, TEXT("GSDK Game client connection port: %d"), GamePorts.ClientConnectionPort);
+            FURL::UrlConfig.DefaultPort = GamePorts.ServerListeningPort;
+            break;
+        }
+    }
+```
+Add above lines after the line below
+```cpp
+UGSDKUtils::RegisterGSDKHealthCheckDelegate(OnGsdkHealthCheck)
+````
+Update <portname> with the name you entered in the port name when creating a build on the MPS dashboard.
+When you use LocalMultiplayerAgent, match <portname> with value of "PortMappingsList.GamePort.Name" in MultiplayerSettings.json file.
+```json
+  "PortMappingsList": [
+    [
+      {
+        "NodePort": 7777,
+        "GamePort": {
+          "Name": "<portname>",
+          "Number": 80,
+          "Protocol": "UTP"
+        }
+      }
+    ]
+  ],
+```
+----
```

Comment:
nit: do we need this dashes?

Reply:
The dashes are section separator at readme.md. 
It would make it easier to read for reader.